packages/jac/src/tests/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './mock-wrapper'
+export { default as ERROR_RESPONSE } from './mocks/fetcher-error-response.json'
```

Comment:
where it used ? this just shows that u didn't really check your changes before sending it for review